### PR TITLE
2022 LUMI docs reorganization

### LINKED ISSUE #40: Storage Space Billing Units
In [computing/jobs/billing](https://docs.lumi-supercomputer.eu/computing/jobs/billing/) it states that 

"for storage, your project is allocated GB-hours."
"Storage is billed by volume as well as time. The billing units are GB-hours."

However in [storage#billing](https://docs.lumi-supercomputer.eu/storage/#billing) it states

"The billing units are TB-hours"

Which is correct? There is conflicting information that needs to be resolved.

### PR DESCRIPTION
**Pull request by Chroxvi**
This implements the new structure of the LUMI documentation as discussed and agreed upon within the LUMI User Support Team. It is the result of the 2022 LUMI docs reorganization handled by @hdrei and @Chroxvi.

### CODE REVIEW

**Comment by egplar** [comment]
I noticed that the URL to the picture in https://docs.lumi-supercomputer.eu/software/packages/quantumespresso/ is broken due to the renaming of the page url. It might be an issue for more pages.

(I put the image in the same directory as the text file, but I noticed that on some other pages with imaging (e.g. distribution and binding) they use pictures from the assets/ directory.)

**Comment by hdrei** [comment]
> I noticed that the URL to the picture in https://docs.lumi-supercomputer.eu/software/packages/quantumespresso/ is broken due to the renaming of the page url. It might be an issue for more pages.
> 
> (I put the image in the same directory as the text file, but I noticed that on some other pages with imaging (e.g. distribution and binding) they use pictures from the assets/ directory.)

I tried to check that all the figures work, and changed some links. Apparently I somehow missed this one! I hope it's the only one...
Edit: To be more precise, I believe it was the only one, as I used the search function in mkdocs to search for all the figures. Unless I made some other error, the rest of the figures should be fine...

File: docs/firststeps/SSH-keys.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    as shown in the figure below. Then select *SSH keys* and click the *New key* button. Now copy and paste the content of your **public** key file in the text area and click the *Add SSH key* button.
+
+    <figure>
+      <img src="../../assets/images/MyAccessID_ssh-key.png" width="480" alt="Screenshot of user profile settings to setup ssh public key">
+      <figcaption>MyAccessID Own profile information to add ssh public key.</figcaption>
+    </figure>
+
+=== "For users with a Finnish allocation"
+
+    Now that you have generated your key pair, you need register your
```
Comment (trz42) [inline docs/firststeps/SSH-keys.md]: minor typo: missing 'to' in 'you need register'

File: docs/firststeps/SSH-keys.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      <img src="../../assets/images/MyAccessID_ssh-key.png" width="480" alt="Screenshot of user profile settings to setup ssh public key">
+      <figcaption>MyAccessID Own profile information to add ssh public key.</figcaption>
+    </figure>
+
+=== "For users with a Finnish allocation"
+
+    Now that you have generated your key pair, you need register your 
+    **public** key in your MyCSC [:material-account: **user profile**][mycsc-profile].
+
+    To register your key with [MyCSC][mycsc-profile], click on *My Profile* item
```
Comment (trz42) [inline docs/firststeps/SSH-keys.md]: minot typo? missing 'the' before *My Profile*?

File: docs/firststeps/SSH-keys.md (hunk 1 above)
Comment (Chroxvi) [reply docs/firststeps/SSH-keys.md]: Fixed in [86d205b](https://github.com/Lumi-supercomputer/lumi-userguide/pull/51/commits/86d205b806a38105172989fb412cf2142c27420a).

File: docs/firststeps/SSH-keys.md (hunk 2 above)
Comment (Chroxvi) [reply docs/firststeps/SSH-keys.md]: Fixed in [86d205b](https://github.com/Lumi-supercomputer/lumi-userguide/pull/51/commits/86d205b806a38105172989fb412cf2142c27420a).

**Review by olouant — APPROVED** [review]

**Pull request merged**